lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -1395,22 +1400,30 @@ protected function generateEntityStubMethod(ClassMetadataInfo $metadata, $type,
     }
 
     /**
-     * @param string            $name
+     * @param string[]          $lifecycleNames
      * @param string            $methodName
      * @param ClassMetadataInfo $metadata
      *
      * @return string
      */
-    protected function generateLifecycleCallbackMethod($name, $methodName, $metadata)
+    protected function generateLifecycleCallbackMethod($lifecycleNames, $methodName, $metadata)
```

Comment:
No, that involves even more chaos later on. I suggest not reusing this method for this new purpose at all...